bazel/README.md

```diff
@@ -201,17 +207,16 @@ for how to update or override dependencies.
     set PATH=%PATH%;%USERPROFILE%\Git\bin
     ```
 
-    Lastly, persist environment variable changes. NOTE: The paths in this document are given as
-    examples, make sure to verify you are using the correct paths for your environment. Also note
-    that these examples assume using a `cmd.exe` shell to set environment variables etc., be sure
-    to do the equivalent if using a different shell.
+    Lastly, persist environment variable changes.
     ```
     setx PATH "%PATH%"
     setx BAZEL_SH "%BAZEL_SH%"
     setx MSYS2_ARG_CONV_EXCL "%MSYS2_ARG_CONV_EXCL%"
     setx BAZEL_VC "%BAZEL_VC%"
     setx TMPDIR "%TMPDIR%"
+    setx MSYS2_PATH_TYPE "%MSYS2_PATH_TYPE%"
     ```
+    > On Windows the supported/recommended shell to interact with bazel is MSYS2. This means that all the bazel commands (i.e. build, test) should be executed from MSYS2.
```

Comment:
one thought: it might be a little confusing we're setting variables in `cmd` but using `msys2 bash`, maybe we should add a note clarifying the two?

Reply:
I see. Do you have a suggestion in mind?

Reply:
Above we state that we are using cmd as an example
>The paths in this document are given as examples, make sure to verify you are using the correct paths for your environment. Also note that these examples assume using a `cmd.exe` shell to set environment variables etc., be sure to do the equivalent if using a different shell.

and here we specify that `MSYS2` is used only to interact with bazel.

Reply:
yeah, maybe its as easy as saying something on the order of "this example shows using `cmd` to set env variables before entering `bash` to ensure environment variables are persisted, you may use `export` from within `bash` to perform the equivalent steps within a `bash` session", idk, we maybe don't need to be so explicit, but we've had a good few issues with build environments

Reply:
oh yeah thats already there i missed that, perfect